public/data/textile/processes.json

```diff
@@ -665,43 +665,6 @@
     "waste": 0,
     "alias": null
   },
-  {
-    "name": "market for electricity, medium voltage, PT",
-    "info": "Energie > Electricité > Mix moyen",
-    "unit": "kWh",
-    "source": "Ecoinvent 3.9.1",
-    "correctif": "",
-    "step_usage": "Energie",
-    "uuid": "b5be40ddf8bb1f731bcaa3cdf9664a49",
-    "impacts": {
-      "acd": 0,
-      "cch": 0,
-      "etf": 0,
-      "etf-c": 0,
-      "fru": 0,
-      "fwe": 0,
-      "htc": 0,
-      "htc-c": 0,
-      "htn": 0,
-      "htn-c": 0,
-      "ior": 0,
-      "ldu": 0,
-      "mru": 0,
-      "ozd": 0,
-      "pco": 0,
-      "pma": 0,
-      "swe": 0,
-      "tre": 0,
-      "wtu": 0,
-      "ecs": 27.708213391614525,
-      "pef": 30.75837108909772
-    },
-    "heat_MJ": 0,
-    "elec_pppm": 0,
-    "elec_MJ": 0,
-    "waste": 0,
-    "alias": null
-  },
```

Comment:
Why is the Portuguese mix removed? And why hasn't it be removed from `processes_impacts.json`?

Reply:
PT elec is not part of the required elec mixes
process_impacts is generated from processes